gslib/commands/rsync.py

```diff
@@ -513,8 +513,14 @@
 
                  will skip all .txt and .jpg files in dir.
 
-                 NOTE: When using this on the Windows command line, use ^ as an
+                 NOTE: When using this in the Windows cmd.exe command line interpreter, use ^ as an
                  escape character instead of \ and escape the | character.
+                 
+                   gsutil rsync -x ".*\.txt$^|.*\.jpg$"
+                   
+                 NOTE: Windows Powershell requires single-quoted regex
+                 
+                   gsutil rsync -x '.*\.txt$^|.*\.jpg$'
```

Comment:
Does Powershell use the same ^ escape character as cmd.exe? If so, my above comment may apply here as well, and it may be worth saying something like: "Windows Powershell requires single-quoted regex and, similar to using the cmd.exe command line interpreter, uses ^ as an escape character"